src/NuGet.Core/NuGet.Packaging/PackageCreation/Resources/NuGetResources.resx

```diff
@@ -208,4 +208,12 @@
   <data name="IconNoFileElement" xml:space="preserve">
     <value>The icon file '{0}' does not exist in the package.</value>
   </data>
+  <data name="Manifest_IconUrlCannotBeUsedWithIcon" xml:space="preserve">
+    <value>The iconUrl and icon elements cannot be used together.</value>
+    <comment>Please, don't localize iconUrl and icon</comment>
+  </data>
+  <data name="IconMissingRequiredValue" xml:space="preserve">
+    <value>The element 'icon' cannot be empty.</value>
```

Comment:
The reference to `icon` element is inconsistent with the message above (there are apostrophes in this message and there are none in the message above).

Reply:
I tried to be consistent with the licenses error messages

Reply:
All resources modified in this PR have the icon/iconUrl between single quotes.